src/main/java/com/github/indigopolecat/bingobrewers/ScoreBoard.java

```diff
@@ -0,0 +1,53 @@
+package com.github.indigopolecat.bingobrewers;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.scoreboard.Score;
+import net.minecraft.scoreboard.ScoreObjective;
+import net.minecraft.scoreboard.ScorePlayerTeam;
+import net.minecraft.scoreboard.Scoreboard;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.EnumChatFormatting;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class ScoreBoard {
+    private static List<String> getScoreboard() {
+        ArrayList<String> scoreboardAsText = new ArrayList<>();
+        if (Minecraft.getMinecraft() == null || Minecraft.getMinecraft().theWorld == null) {
+            return scoreboardAsText;
+        }
+        Scoreboard scoreboard = Minecraft.getMinecraft().theWorld.getScoreboard();
+        ScoreObjective sideBarObjective = scoreboard.getObjectiveInDisplaySlot(1);
+        if (sideBarObjective == null) {
+            return scoreboardAsText;
+        }
+        String scoreboardTitle = sideBarObjective.getDisplayName();
+        scoreboardTitle = EnumChatFormatting.getTextWithoutFormattingCodes(scoreboardTitle);
+        scoreboardAsText.add(scoreboardTitle);
+        Collection<Score> scoreboardLines = scoreboard.getSortedScores(sideBarObjective);
+        for (Score line : scoreboardLines) {
+            String playerName = line.getPlayerName();
+            if (playerName == null || playerName.startsWith("#")) {
+                continue;
+            }
+            ScorePlayerTeam scorePlayerTeam = scoreboard.getPlayersTeam(playerName);
+            String lineText = EnumChatFormatting.getTextWithoutFormattingCodes(
+                    ScorePlayerTeam.formatPlayerName(scorePlayerTeam, line.getPlayerName()));
+            scoreboardAsText.add(lineText.replace(line.getPlayerName(), ""));
+        }
+        return scoreboardAsText;
+    }
+
+    public static boolean isBingo() {
+        boolean bingo = false;
+        List<String> scoreBoardLines = getScoreboard();
+        int size = scoreBoardLines.size() - 1;
+        for (int i = 0; i < scoreBoardLines.size(); i++) {
+            String line = EnumChatFormatting.getTextWithoutFormattingCodes(scoreBoardLines.get(size - i).toLowerCase());
+            if(line.contains("bingo")) bingo = true;
+        }
+        return bingo;
+    }
```

Comment:
The PR lgtm, i would just adjust that we instand return true as soon "bingo" is found here
```suggestion
        List<String> scoreBoardLines = getScoreboard();
        int size = scoreBoardLines.size() - 1;
        for (int i = 0; i < scoreBoardLines.size(); i++) {
            String line = EnumChatFormatting.getTextWithoutFormattingCodes(scoreBoardLines.get(size - i).toLowerCase());
            if(line.contains("bingo")) return true;
        }
        return false;
    }
```